hic_qc.py

```diff
@@ -921,6 +948,10 @@ def stringify_stats(self):
                             'many_zero_dist_threshold': (100.0 * self.max_zero_dist_percentage, '{}'),
                             'many_zero_mapq_threshold': (100.0 * self.max_zero_mapq0_percentage, '{}'),
                             'many_unmapped_threshold': (100.0 * self.max_unmapped_percentage, '{}'),
+                            'alignment_command_line': (self.bwa_command, '{}'),
+                            'samblaster': (self.samblaster, '{}'),
+                            'lib_enzyme': (''.join(self.lib_enzyme), '{}'),
```

Comment:
This seems like you would end up with 's3m1' as the string if two enzymes are specified making self.lib_enzyme a list like ['s3', 'm1'].  This will make the 'undecided' and single enzyme values work, but num_enzymes > 1 will just be a spaceless string of the names used.  Is this intended?

Reply:
Changing it to ', ' and adjusting the default arg so it's a list too, so self.lib_enzyme is always a list